extensions/ql-vscode/src/pure/interface-types.ts

```diff
@@ -53,9 +61,11 @@ export interface Interpretation {
    * they appear in the sarif file.
    */
   sortState?: InterpretedResultsSortState;
```

Comment:
Does this field make sense for graph results?

Reply:
No, so I'll move it into `SarifInterpretationData`.